cypher-shell/src/main/java/org/neo4j/shell/commands/Use.java

```diff
@@ -0,0 +1,60 @@
+package org.neo4j.shell.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nonnull;
+
+import org.neo4j.shell.DatabaseManager;
+import org.neo4j.shell.exception.CommandException;
+import org.neo4j.shell.exception.ExitException;
+
+import static org.neo4j.shell.commands.CommandHelper.simpleArgParse;
+
+/**
+ * This command starts a transaction.
+ */
+public class Use implements Command {
+    private static final String COMMAND_NAME = ":use";
+    @Nonnull private final DatabaseManager databaseManager;
+    @Nonnull private String databaseName;
+
+    public Use(@Nonnull final DatabaseManager databaseManager) {
+        this.databaseManager = databaseManager;
+    }
+
+    @Nonnull
+    @Override
+    public String getName() {
+        return COMMAND_NAME;
+    }
+
+    @Nonnull
+    @Override
+    public String getDescription() {
+        return "Set the active database";
+    }
+
+    @Nonnull
+    @Override
+    public String getUsage() {
+        return "database";
+    }
+
+    @Nonnull
+    @Override
+    public String getHelp() {
+        return String.format("Set the active database that transactions are executed on", Commit.COMMAND_NAME);
```

Comment:
Sets?

Reply:
No, this is meant to be in the imperative mood. Going through our commands I see now that we have a mixture of imperative mood (slight overhead) and third-person-whateveritscalled. Maybe we should make it consistent and a conscious policy. I personally prefer the imperative mood, and it seem to be the standard for `man` pages of commands as well as a popular git commit message style guide.